### PR TITLE
IMUtils: new recipe

### PR DESCRIPTION
**Pull request by extrowerk**

### CODE REVIEW

File: dev-python/imutils/imutils-0.5.3.recipe [hunk 1]
```diff
@@ -0,0 +1,78 @@
...
+for i in "${!PYTHON_PACKAGES[@]}"; do
+pythonPackage=${PYTHON_PACKAGES[i]}
+pythonVersion=${PYTHON_VERSIONS[$i]}
+eval "PROVIDES_${pythonPackage}=\"\
+	${portName}_$pythonPackage = $portVersion\n\
+	cmd:range_detector$pythonVersion\n\
+	\"; \
+REQUIRES_$pythonPackage=\"\
+	haiku$secondaryArchSuffix\n\
+	cmd:cython$pythonVersion\n\
```
Comment (Begasus) [inline dev-python/imutils/imutils-0.5.3.recipe:42]: Same thing here as for numpy:
```diff
- 	cmd:cython$pythonVersion\n\
+ 	cmd:cython$pythonVersion$secondaryArchSuffix\n\
```

File: dev-python/imutils/imutils-0.5.3.recipe [hunk 2]
```diff
@@ -0,0 +1,78 @@
...
+	cmd:python$pythonVersion\
+	\""
+BUILD_REQUIRES="$BUILD_REQUIRES
+	setuptools_$pythonPackage
+	"
+BUILD_PREREQUIRES="$BUILD_PREREQUIRES
+	cmd:cython$pythonVersion
```
Comment (Begasus) [inline dev-python/imutils/imutils-0.5.3.recipe:49]:
```diff
- 	cmd:cython$pythonVersion
+ 	cmd:cython$pythonVersion$secondaryArchSuffix
```

File: dev-python/imutils/imutils-0.5.3.recipe [hunk 3]
```diff
@@ -0,0 +1,78 @@
...
+		pythonVersion=${PYTHON_VERSIONS[$i]}
+
+		python=python$pythonVersion
+		installLocation=$prefix/lib/$python/vendor-packages/
+		export PYTHONPATH=$installLocation:$PYTHONPATH
+		mkdir -p $installLocation
+		rm -rf build
+		$python setup.py build install \
+			--root=/ --prefix=$prefix
+		mv $binDir/range-detector $binDir/range_detector$pythonVersion
```
Comment (Begasus) [inline dev-python/imutils/imutils-0.5.3.recipe:72]: binary is installed in $prefix/bin so:
```diff
- 		mv $binDir/range-detector $binDir/range_detector$pythonVersion
+ 		mv $prefix/bin/range-detector $prefix/bin/range_detector$pythonVersion
```

**Review by Begasus — CHANGES_REQUESTED** [review]

**Comment by korli** [comment]
Looks like a stale project (no commit for a year despite a pile of pull requests), not widely available:
https://repology.org/project/python:imutils/versions

**Comment by extrowerk** [comment]
@Begasus: thanks, fixed.
@korli: i know, still it is used by some opencv related projects.

**Pull request merged**